examples/source/1.components/amp-analytics/index.html

```diff
@@ -54,66 +46,6 @@
       custom-element="amp-iframe"
       src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"
     ></script>
-    <link rel="canonical" href="<% canonical %>" />
-    <meta name="viewport" content="width=device-width" />
```

Comment:
Can we remove viewport and boilerplate if optimizer is not running on this page?

Reply:
they are redundant - sorry, I didn't include proper context in the description.

The contents of this file are being included inside of [the page](https://amp.dev/documentation/examples/components/amp-analytics/?referrer=ampbyexample.com), via [this template](https://github.com/ampproject/amp.dev/blob/future/frontend/templates/layouts/default.j2). These tags are being included on the rendered version of this page [here](https://github.com/ampproject/amp.dev/blob/future/frontend/templates/layouts/default.j2#L50) and [here](https://github.com/ampproject/amp.dev/blob/future/frontend/templates/layouts/default.j2#L73) specifically

Reply:
What about the standalone playground versions of the examples?

Reply:
ah - I figured this was too silly to work.

Assuming things were working "correctly" - how would the duplicate tags (i.e. `<meta charset="utf-8">`) be expected to be removed?

Reply:
The playground pulls in the original sample file.

Reply:
I understand that now, I am asking in what way is the link @CrystalOnScript provided - `https://amp.dev/documentation/examples/components/amp-analytics` expected to include that example in a way that is a valid document?

Reply:
Let's focus on this once we have a working build and the outstanding pixie checks are done.

Reply:
sgtm

Reply:
@sebastianbenz if you would be able to answer the previous question, that would be great! thanks

Reply:
The components should be imported automatically via a grow extension. The boilerplate will be removed and normalized in samplesBuilder.js.